internal/receiver/scriptedinputsreceiver/procpipe/commander.go

```diff
@@ -0,0 +1,155 @@
+// Copyright Copyright Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package procpipe
+
+import (
+	"context"
+	"os/exec"
+	"strings"
+	"sync/atomic"
+	"syscall"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+// Start the shell and begin watching the process.
+// shell's stdout and stderr are written to a file.
+func (c *Commander) Start(ctx context.Context) error {
+	c.logger.Info("Starting script", zap.String("script", c.execFilePath))
+
+	stdout, _ := exec.Command("which", "sh").Output()
```

Comment:
the splunk-otel-collector image doesn't provide `which`. Can we just use `sh` and address any required updates in a separate PR? `exec` uses https://pkg.go.dev/os/exec#LookPath by default.